src/ceph-volume/ceph_volume/util/encryption.py

```diff
@@ -18,8 +19,18 @@ def set_dmcrypt_no_workqueue(target_version: str = '2.3.4') -> None:
     """
     command = ["cryptsetup", "--version"]
     out, err, rc = process.call(command)
+    pattern: str = r'\b\d+(\.\d+)*\b'
+
+    if rc:
+        raise RuntimeError("Can't retrieve cryptsetup version.")
```

Comment:
Print the underlaying error? `RuntimeError` can be a lot of things.

Reply:
added `err` to be printed here

Reply:
cimer maggle